docs/docs/governance/cross-chain-governance.md

```diff
@@ -0,0 +1,134 @@
+# Cross Chain Governance
+
+To simplify the maintenance and management of the protocol contracts, we use a cross-chain governance that allows for a DAO proposal on mainnet to reach contracts on other networks.
+
+# How it works
+
+To reach other chains, calls emitted from the mainnet DAO go though the [Connext bridge](https://www.connext.network/) and are executed on the other side of the bridge, after a period of cooldown.
+
+```
+(mainnet)                       (destination chain)    (cooldown period)
+DAO proposal >  Connext Bridge >   Safe multisig    >  wait for 7 days   >
+```
+
+The workflow is as follow
+
+1. a DAO proposal is created, containing 1 call per chain
+2. if the vote succeed, the DAO proposal is executed. All calls are send to the bridge
+3. each call cross the bridge seperately towards its destination on a specific chain
+4. the call is received on the destination chain by Unlock multisig
+5. once received, the call is held in the multisig for a period of X days during which it can be cancelled.
+6. once the cooldown period ends, the call is executed
+
+NB: The cooldown period is useful to prevent malicious or errored calls from being executed
```

Comment:
```suggestion
NB: The cooldown period is useful to prevent malicious or errored calls from being executed if the bridge itself has been compromised.
```